x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/conversations/conversations.spec.ts

```diff
@@ -824,5 +822,65 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
         expect(ownerDeleteResponse.status).to.be(200);
       });
     });
+
+    describe('conversation archiving', () => {
+      let createdConversationId: string;
+
+      before(async () => {
+        const { status, body } = await observabilityAIAssistantAPIClient.editor({
+          endpoint: 'POST /internal/observability_ai_assistant/conversation',
+          params: {
+            body: {
+              conversation: conversationCreate,
+            },
+          },
+        });
+
+        expect(status).to.be(200);
+        createdConversationId = body.conversation.id;
```

Comment:
I see we repeat this for every test. Perhaps an helper would be useful

```suggestion
        const { id } = await createConversation(conversationCreateBody)
        createdConversationId = id;
```

Reply:
Done